### ISSUE TITLE
use-json-log; uses Error Stream on Windows instead to log to logfile

### ISSUE THREAD

**Issue by Kriegel** [rclone/rclone#4367]
Hi There!

If I use the following command, the JSON Messages go to the screen but logfile stays empty.
```
rclone.exe copy "C:\temp\Source" "C:\temp\Destination" --use-json-log -v --log-file="C:\temp\Destination\Rclone.log"
``` 
With the help of PowerShell I noticed that all Text was displayed RED

So If I redirect the Error Stream to a file I get the Messages into the file. Like desired
See code here:
```
rclone.exe copy "C:\temp\Source" "C:\temp\Destination" --use-json-log -v > "C:\temp\Destination\Rclone.log" 2>&1
``` 
Attached a full -vv Log (catch with the error Stream 2>&1 )
[Rclone.txt](https://github.com/rclone/rclone/files/4794998/Rclone.txt)

**Comment by ncw** [comment]
I've managed to confirm that this is a bug - thanks for reporting.

Can you try this fix?

https://beta.rclone.org/branch/v1.52.2-133-g683ff312-fix-4367-logrus-beta/ (uploaded in 15-30 mins)

**Comment by Kriegel** [comment]
Sorry i, am bussy, and because of the bad Windows support of rclone quitted to use it.

**Comment by ncw** [comment]
> Sorry i, am bussy, and because of the bad Windows support of rclone, I quitted to use it.

Sorry to hear that - what was the problem?

I've merged this fix to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.53 or maybe 1.52.3 depending...
